app/utils/chart-config/api.ts

```diff
@@ -10,6 +13,7 @@ export const createConfig = async (state: ConfiguratorStatePublishing) => {
   return apiFetch<InferAPIResponse<typeof apiConfigs, "POST">>("/api/config", {
     method: "POST",
     data: {
+      isDraft: true,
```

Comment:
Why? Shouldn't this depend on whether we publish or not?

Reply:
Create is the first time. It's update that will be used for publish.